AdyenComponents/UPI/UPIComponent.swift

```diff
@@ -215,6 +304,22 @@ public final class UPIComponent: PaymentComponent,
         }
     }
 
+    private func changeUPIAppsListVisiblity(shouldHide isHidden: Bool = false) {
+        if isHidden {
```

Comment:
This is very nitpicky so feel free to ignore but what about simplifying the code to the following
```
guard !upiPaymentMethod.apps.isEmpty else { return }

upiAppsList.forEach {
    $0.isHidden.wrappedValue = isHidden
}
```

Reply:
I think this should be addressed.

Reply:
As a sidenote:
We use `!upiPaymentMethod.apps.isEmpty` a lot in the code. Maybe we can create a meaningful property variable for this?

Reply:
This I am going to check but in next PR.